lib/catalog/martyrology.ts

```diff
@@ -1893,6 +2036,19 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       canonizationLevel: CanonizationLevels.Saint,
       name: 'Jacinta Marto',
     },
+    // src:
+    // - mr_fr_1982_ed2_coutances
+    // - https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/bienheureux-francois-lefranc-et-les-martyrs-de-la-revolution
+    // - https://en.wikipedia.org/wiki/Jacques-Fran%C3%A7ois_Lefranc
+    jacques_francois_lefranc_martyr: {
+      canonizationLevel: CanonizationLevels.Blessed,
+      name: 'Jacques François Lefranc',
+      titles: [Title.Martyr],
+      dateOfBirth: '1739-03-30',
+      dateOfDeath: {
+        between: ['1792-09-02', '1792-09-03'],
```

Comment:
Ok, I think you're right. In that case, it's probably better to create a separate entity for the martyrs of the revolution, to avoid losing precision.

Reply:
> it's probably better to create a separate entity for the martyrs of the revolution, to avoid losing precision.

:100: yes. :wink: